ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -99,7 +87,7 @@ export class TransactionDetailsScreen extends React.Component<Props, State> {
    * then the "Thank you message" is displayed
    */
   private getSubHeader() {
-    return this.state.isTransactionStarted ? (
+    return this.props.navigation.getParam("paymentCompleted", false) ? (
```

Comment:
improve readability here

```ts
const isPaymentCompleted = this.props.navigation.getParam("paymentCompleted", false);
return isPaymentCompleted ? () : ();
```

Reply:
event more readable would be having two sub-components and the parent one displaying either based on the prop